src/builder.rs

```diff
@@ -1492,6 +1564,196 @@ where
+                    && (matches!(agg_numeric_type.aggregate_op, AggregateOperation::Min)
+                        || matches!(agg_numeric_type.aggregate_op, AggregateOperation::Max))
+                {
+                    // Check if this is a UUID field by looking at the table columns
+                    for col in agg_numeric_type.table.columns.iter() {
+                        let column_name = &col.name;
+                        if col_name == column_name
+                            || col_name.to_lowercase() == column_name.to_lowercase()
+                        {
+                            if let Some(ref type_) = col.type_ {
+                                if type_.name == "uuid" {
+                                    return Err(format!(
+                                        "UUID fields (like \"{}\") are not supported for min/max aggregation because they don't have a meaningful natural ordering",
+                                        col_name
+                                    ));
+                                }
+                            }
+                        }
+                    }
+                }
+
+                // Regular error if field not found
+                let sub_field = sub_field.ok_or(format!(
+                    "Unknown field \"{}\" selected on type \"{}\"",
+                    col_name,
+                    type_.name().unwrap_or_default()
+                ))?;
+
+                // Ensure the selected field is actually a column
+                let __Type::Scalar(_) = sub_field.type_().unmodified_type() else {
+                    return Err(format!(
+                        "Field \"{}\" on type \"{}\" is not a scalar column",
+                        col_name,
+                        type_.name().unwrap_or_default()
+                    ));
+                };
+                // We expect the sql_type to be set for columns within the numeric aggregate type's fields
+                // This might require adjustment in how AggregateNumericType fields are created in graphql.rs if sql_type isn't populated there
+                let Some(NodeSQLType::Column(column)) = sub_field.sql_type.clone() else {
+                    // We need the Arc<Column>! It should be available via the __Field's sql_type.
+                    // If it's not, the creation of AggregateNumericType fields in graphql.rs needs adjustment.
+                    return Err(format!(
+                        "Internal error: Missing column info for aggregate field '{}'",
+                        col_name
+                    ));
+                };
+
+                let alias = col_field
+                    .alias
+                    .as_ref()
+                    .map_or(col_name, |x| x.as_ref())
+                    .to_string();
+
+                col_selections.push(ColumnBuilder { alias, column });
+            }
+            Selection::FragmentSpread(_) | Selection::InlineFragment(_) => {
```

Comment:
Consider calling the `normalize_selection_set` function to support fragments.

Reply:
Fixed